src/sounds.cpp

```diff
@@ -283,11 +283,8 @@ void sounds::process_sound_markers( player *p )
         }
         // Only print a description if it exists
         if( !description.empty() ) {
-            // If it came from us, don't print a direction
             if( pos == p->pos() ) {
-                std::string uppercased = description;
-                capitalize_letter( uppercased, 0 );
-                add_msg( "%s", uppercased.c_str() );
+                add_msg( "You hear %s", description.c_str());
```

Comment:
This branch is taken when the sound comes from the very same tile as the player is currently standing on. It is *not* taken when the sound comes from below them.

Some places explicitly assume that there will be no prefix if player location is the sound location and show a different message.

Sitting in a car with a stereo now shows "You hear You listen to a sweet guitar solo!"

Also: standing on top of the car (directly over the sound source) already prints "From the below you hear a sweet guitar solo!"

Can anyone still confirm the original bug (in the current experimental version, the bug report does not list a version).